frontend/constants/sizes.ts

```diff
@@ -0,0 +1,2 @@
+export const APP_WIDTH = 460;
+export const MODAL_WIDTH = APP_WIDTH - 60;
```

Comment:
this is very useful, thanks 👍 could you please use it in `FirstRunModal` (in `frontend/components/Main/MainHeader.tsx`) for consistency